### ISSUE TITLE
Not support Type extFAT filesystem SD card!

### ISSUE THREAD

**Issue by qiurigao** [LineageOS/android_device_bq_tenshi#1]
![screenshot_19700103-104925](https://cloud.githubusercontent.com/assets/22121479/26394685/528b8d84-40a0-11e7-8650-9d7ee85b3509.png)

**Comment by brinlyau** [comment]
Yeah, that's intended. Use another format for your card (mine is fat32). We don't accept bugs reports through here btw!
